src/reactapp/src/api/cart/restSetGuestPaymentMethod/modifier.js

```diff
@@ -0,0 +1,6 @@
+export default function restSetGuestPaymentMethodModifier(result) {
+  return {
+    order_number: result,
+    redirectUrl: '/payone/onepage/redirect/',
```

Comment:
@progammer-rkt should a payone url live in this repo?

Reply:
@wigman good catch. this sould not be there in the rep. i removed it now :)